### ISSUE TITLE
gaphor won't start: importerror in gi.respository

### ISSUE THREAD

**Issue by JensPfeifle** [gaphor/gaphor#66]
### I'm submitting a...
- [x] Bug report  
- [ ] Feature request
- [ ] Documentation issue or request

### Current behavior
Gaphor exits after an ImportError is raised from gi:
```bash
raceback (most recent call last):
  File "/usr/local/bin/gaphor", line 11, in <module>
    load_entry_point('gaphor==0.17.2', 'console_scripts', 'gaphor')()
  File "/usr/local/lib/python2.7/dist-packages/gaphor/__init__.py", line 98, in main
    launch(model)
  File "/usr/local/lib/python2.7/dist-packages/gaphor/__init__.py", line 34, in launch
    Application.init()
  File "/usr/local/lib/python2.7/dist-packages/gaphor/application.py", line 53, in init
    self.load_services(services)
  File "/usr/local/lib/python2.7/dist-packages/gaphor/application.py", line 75, in load_services
    cls = ep.load()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/usr/lib/python2.7/dist-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/usr/local/lib/python2.7/dist-packages/gaphor/services/filemanager.py", line 13, in <module>
    from gaphor.storage import storage, verify
  File "/usr/local/lib/python2.7/dist-packages/gaphor/storage/storage.py", line 18, in <module>
    import gaphas
  File "/usr/local/lib/python2.7/dist-packages/gaphas/__init__.py", line 45, in <module>
    from .canvas import Canvas
  File "/usr/local/lib/python2.7/dist-packages/gaphas/canvas.py", line 43, in <module>
    from gaphas.decorators import nonrecursive, AsyncIO
  File "/usr/local/lib/python2.7/dist-packages/gaphas/decorators.py", line 9, in <module>
    import gi
  File "/usr/lib/python2.7/dist-packages/gi/__init__.py", line 39, in <module>
    raise ImportError(_static_binding_error)
ImportError: When using gi.repository you must not import static modules like "gobject". Please change all occurrences of "import gobject" to "from gi.repository import GObject". See: https://bugzilla.gnome.org/show_bug.cgi?id=709183
```

### Expected behavior
gaphor starts.

### Minimal reproduction of the problem with instructions
Install gaphor via pip: `python2 -m pip install gaphor`

### System Information
OS:
- [x] Linux (Please put in notes the specific distro)
- [ ] MacOS
- [ ] Windows 10
- [ ] Windows 7

NOTES:

Python: 
- 2.7.15rc1

System:    
- Kernel: 4.15.0-43-generic x86_64 bits: 64 
- Desktop: Cinnamon 3.8.9
- Distro: Linux Mint 19 Tara

**Comment by danyeaw** [comment]
Hi @JensPfeifle, thanks for your contribution by submitting this issue :heart: 

I think the issue may be that when you `pip install gaphor`, you are pulling in the latest version of `gaphas` which is 1.0.0. This released version of gaphas is compatible with Python3 and uses PyGObject, but the latest version of gaphor uses PyGTK. I am busy working on getting gaphor upgraded as well, and I'm getting very close.

Can you try to run `python2 -m pip install -I gaphas==0.7.2`? This will pin the gaphas version to the previous version that uses PyGTK as well.

**Comment by JensPfeifle** [comment]
Hi Dan, 

I uninstalled `gaphor` and `gaphas` and reinstalled both, pinning the `gaphas` version as you suggested. 

Working now with:
`gaphas                        0.7.2`
`gaphor                        0.17.2`

Maybe it would be good to add a note to the docs [here](https://gaphor.readthedocs.io/en/latest/linux.html)?

Thanks for your help!

**Comment by danyeaw** [comment]
@JensPfeifle I am glad that fixed it :+1: 
Would you be interested in submitting a PR to update the README?

**Comment by JensPfeifle** [comment]
@danyeaw Sure, I can add a note. In the README or in the [documentation](https://gaphor.readthedocs.io/en/latest/linux.html)?

Something like the following?
```
Note:
Gaphor is currently incompatible with versions of gaphas newer than 0.7.2.
Newer versions of gaphas use PyGObject, but gaphor uses PyGTK.

Therefore, gaphas must be pinned to version 0.7.2 with
python2 -m pip install -I gaphas==0.7.2

Make sure to uninstall gaphas again before pinning the version 
if a newer version is already installed in your environment.
```

I'm not very familiar with python packaging, but would it also be possible to set  the requirements in [setup.py](https://github.com/gaphor/gaphor/blob/9a55a28fa0b5e7a7415f74b846ca90c46b27a475/setup.py#L76) to prevent pip from installing a newer version of gaphas?
i.e. [line 76](https://github.com/gaphor/gaphor/blob/9a55a28fa0b5e7a7415f74b846ca90c46b27a475/setup.py#L76) from `"gaphas >= 0.7.2"` to `"gaphas = 0.7.2"`

**Comment by danyeaw** [comment]
@JensPfeifle That note sounds awesome, thanks! I think users would normally check the README for install instructions.
Yes, I agree that changing setup.py and then packaging and releasing a new version of gaphor would be the ideal fix. However, this would involve some significant work, and I rather spend the time getting a PyGObject and Python 3 compatible version of gaphor finished. :+1:

**Comment by danyeaw** [comment]
Closed by #67
